packages/textfield/src/spectrum-config.js

```diff
@@ -65,6 +65,11 @@ const config = {
                     selector: '.is-disabled',
                     name: 'disabled',
                 },
+                {
```

Comment:
`.is-password` is not a part of the Spectrum CSS styling API so it won't need to be a part of this config.